src/gevent_server.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf8 -*-
+from __future__ import unicode_literals
+from server import response_error
+from server import response_ok
+from server import resolve_uri
+from server import parse_request
+
+
+def server(conn, address):
+    """Returns messages to client."""
+    try:
+        while True:
+            conn, addr = server.accept()
```

Comment:
You don't need this outer loop at all.  The `conn` and `address` that are handed to this function as parameters are in fact the ones that you are trying to create here.  You need not re-create them.  strip out the outer try...except and the `while True` loop and what remains is what you need.  The rest is provided by the gevent StreamServer